pepper/templates/emails/application_decisions/waitlisted.html

```diff
@@ -2,42 +2,41 @@
 {% block message %}
     <tr>
       <td class="content-block">
-        Hi {{ user.fname }} {{ user.lname }},
+        Hey {{ user.fname }},
       </td>
     </tr>
     <tr>
       <td class="content-block">
-        Thank you for taking the time to apply to {{ config.HACKATHON_NAME }}. Your application has been waitlisted.
+        Thank you for taking the time to apply to {{ config.EVENT_NAME }}. Unfortunately, we had many more applicants than we have 
+        space for, and we are unable to offer you a spot for this year. However, we're still hoping that we can fit you in, so we 
+        have made you eligible for our <strong>day-of waitlist</strong>.
```

Comment:
Suggested changes to reflect waitlist with a possibility of later admission:
 
Thank you for taking the time to apply to {{ config.EVENT_NAME }}. Unfortunately, we had many more applicants than we have space for and we are unable to offer you a spot at this moment. However, there is a good chance we can still find a space for you. Acceptances are released in waves, so we will let you know in a few days if a spot opens up!

Reply:
though what is currently here is good for our final waitlist message.